src/control/server/testdata/expect_daos_server_sockets.yml

```diff
@@ -2,7 +2,7 @@ name: daos_server
 servers:
 - rank: null
   targets: 8
-  nr_xs_helpers: 2
+  nr_xs_helpers: 8
```

Comment:
I don't know what these .yml files are used for but it seems safest to set nr_xs_helpers in such a way that there would be no behavior change. so suggest setting to 16 here (if I understand the new approach)?